Makefile

```diff
@@ -27,13 +27,16 @@ all: generate controller verify
 controller:
 	$(MAKE) -f kubebuilder.mk manager
 
-# Run generators for protos, Deepcopy funcs, CRDs, and docs..
+# Run generators for protos, Deepcopy funcs, CRDs, and docs.
+#
+# Order here matters; we need to generate Go code before generating
+# protobuf. Generating proto is really slow, so it's last.
 .PHONY: generate
 generate:
-	$(MAKE) proto
 	$(MAKE) -f kubebuilder.mk generate
 	$(MAKE) manifests
 	$(MAKE) docs
+	$(MAKE) proto
```

Comment:
This seems like it might be out of scope for this PR?

Reply:
I can split this into a separate PR if you prefer?

Reply:
Moved into #267